### PR TITLE
Creating Book Model

### PR DESCRIPTION
**Pull request by torsello**
## Summary
- Book model created
- Book table created from model

## Feature screenshot
- Whitelabel on http://localhost:8081/
https://i.imgur.com/a5rvnFp.png

- Book table generated
https://i.imgur.com/JfrpS3S.png

## Trello Card
https://trello.com/c/yqrhGHHG

### CODE REVIEW

**Review by julianvasquez-wolox — APPROVED** [review]

File: src/main/java/com/wolox/training/models/Book.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @Column(nullable = true)
```
Comment (zeminlu) [inline src/main/java/com/wolox/training/models/Book.java]: If you're only going to define the non-nullability, don't use @Column but instead use @NotNull. Change it for every case.
If variable is nullable, just leave it without annotations.
Also, leave a blank line between variables.

File: src/main/java/com/wolox/training/models/Book.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+    public Book() {
+
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
```
Comment (zeminlu) [inline src/main/java/com/wolox/training/models/Book.java]: The id setter shouldn't be defined unless exctrictly necessary and justified, given these ids will be sorted out by JPA.

**Review by zeminlu — CHANGES_REQUESTED** [review]
Looks good! Check my comments.

File: src/main/java/com/wolox/training/models/Book.java (hunk 1 above)
Comment (torsello) [reply src/main/java/com/wolox/training/models/Book.java]: Thanks, that's a good tip!

File: src/main/java/com/wolox/training/models/Book.java (hunk 2 above)
Comment (torsello) [reply src/main/java/com/wolox/training/models/Book.java]: Perfect, I start with the changes :+1:

**Comment by torsello** [comment]
@zeminlu changes made :+1:

File: build.gradle [hunk 3]
```diff
@@ -17,6 +17,7 @@ dependencies {
 	compile('org.springframework.boot:spring-boot-starter-web')
 	compile group: 'org.postgresql', name: 'postgresql', version: '42.2.5'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	compile group: 'org.hibernate', name: 'hibernate-validator', version: '6.1.0.Final'
```
Comment (zeminlu) [inline build.gradle]: You should use `spring-boot-starter-validation` dependency.

**Review by zeminlu — CHANGES_REQUESTED** [review]
Looks good! Just a small thing and done!

**Review by zeminlu — APPROVED** [review]

**Pull request merged**